twoopstracker/authentication/urls.py

```diff
@@ -11,6 +13,10 @@
         name="account_email_verification_sent",
     ),
     path("google/", GoogleLogin.as_view(), name="google_login"),
+    path(
+        "registration/account-confirm-email/<str:key>/",
+        RedirectView.as_view(url=f"{settings.TWOOPSTRACKER_CONFIRM_EMAIL_URL}/%(key)s"),
```

Comment:
What is this `%(key)s` magic?

Reply:
@kilemensi its just another [string formatting](https://docs.python.org/3/library/stdtypes.html#str.format) placeholder for named arguments